benchmarks/structured/qft/qft.qasm

```diff
@@ -0,0 +1,19 @@
+OPENQASM 3.0;
+include "qelib1.inc";
+
+input int n;
+
+qubit[n] q;
+bit[n] c;
+
+for int i in [0:n-1] {
+    h q[i];
+    for int j in [i+1:n-1] {
+        ctrl @ p(pi/2**(j-i)) q[j], q[i];
+    }
+}
+for int i in [0:(n-1)/2] {
+    swap q[i], q[n - 1 - i];
```

Comment:
Not sure how/if the spec defines it, but some libraries error for same index swap / `swap q[1], q[1]`